### PR TITLE
Math.xml samples -- updating for Try .NET

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
docs PR is https://github.com/dotnet/dotnet-api-docs/pull/3731

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CLR/math.max/CS/max.cs [hunk 1]
```diff
@@ -1,62 +1,63 @@
...
+
+        Console.WriteLine("{0}Display the greater of two values:{0}", nl);
+        Console.WriteLine(str, "Byte   ", xByte1, xByte2, Math.Max(xByte1, xByte2));
+        Console.WriteLine(str, "Int16  ", xShort1, xShort2, Math.Max(xShort1, xShort2));
+        Console.WriteLine(str, "Int32  ", xInt1, xInt2, Math.Max(xInt1, xInt2));
+        Console.WriteLine(str, "Int64  ", xLong1, xLong2, Math.Max(xLong1, xLong2));
+        Console.WriteLine(str, "Single ", xSingle1, xSingle2, Math.Max(xSingle1, xSingle2));
+        Console.WriteLine(str, "Double ", xDouble1, xDouble2, Math.Max(xDouble1, xDouble2));
+        Console.WriteLine(str, "Decimal", xDecimal1, xDecimal2, Math.Max(xDecimal1, xDecimal2));
+        //
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CLR/math.max/CS/max.cs]: Empty comment.

```suggestion

```

File: snippets/csharp/VS_Snippets_CLR/math.min/CS/min.cs [hunk 2]
```diff
@@ -1,62 +1,64 @@
...
+        Console.WriteLine("{0}Display the lesser of two values:{0}", nl);
+        Console.WriteLine(str, "Byte   ", xByte1, xByte2, Math.Min(xByte1, xByte2));
+        Console.WriteLine(str, "Int16  ", xShort1, xShort2, Math.Min(xShort1, xShort2));
+        Console.WriteLine(str, "Int32  ", xInt1, xInt2, Math.Min(xInt1, xInt2));
+        Console.WriteLine(str, "Int64  ", xLong1, xLong2, Math.Min(xLong1, xLong2));
+        Console.WriteLine(str, "Single ", xSingle1, xSingle2, Math.Min(xSingle1, xSingle2));
+        Console.WriteLine(str, "Double ", xDouble1, xDouble2, Math.Min(xDouble1, xDouble2));
+        Console.WriteLine(str, "Decimal", xDecimal1, xDecimal2, Math.Min(xDecimal1, xDecimal2));
+     
+        //
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CLR/math.min/CS/min.cs]:
```suggestion

```

File: snippets/csharp/VS_Snippets_CLR/math.min/CS/min.cs [hunk 3]
```diff
@@ -1,62 +1,64 @@
...
-    Console.WriteLine(str, "Decimal", xDecimal1, xDecimal2, Math.Min(xDecimal1, xDecimal2));
-//
-    Console.WriteLine("{0}The following types are not CLS-compliant:{0}", nl);
-    Console.WriteLine(str, "SByte  ", xSbyte1, xSbyte2, Math.Min(xSbyte1, xSbyte2));
-    Console.WriteLine(str, "UInt16 ", xUshort1, xUshort2, Math.Min(xUshort1, xUshort2));
-    Console.WriteLine(str, "UInt32 ", xUint1, xUint2, Math.Min(xUint1, xUint2));
-    Console.WriteLine(str, "UInt64 ", xUlong1, xUlong2, Math.Min(xUlong1, xUlong2));
+        //<snippet1>
+        string str = "{0}: The lesser of {1,3} and {2,3} is {3}.";
+        string nl = Environment.NewLine;
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CLR/math.min/CS/min.cs]: Why not delete this and just include \n in the Console.WriteLine statements. Using \n directly seems more cleaner to me.

File: snippets/csharp/VS_Snippets_CLR_System/system.Math.Abs/cs/Abs1.cs [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
 
 public class Example
 {
    public static void Main()
    {
+      // <Snippet1>
       decimal[] decimals = { Decimal.MaxValue, 12.45M, 0M, -19.69M, 
                              Decimal.MinValue };
       foreach (decimal value in decimals)
          Console.WriteLine("Abs({0}) = {1}", value, Math.Abs(value));
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_CLR_System/system.Math.Abs/cs/Abs1.cs]: You may check if there are other places where string interpolation can be used.

```suggestion
         Console.WriteLine($"Abs({value}) = {Math.Abs(value)}");
```

File: snippets/csharp/VS_Snippets_CLR/math.min/CS/min.cs (hunk 3 above)
Comment (BillWagner) [reply snippets/csharp/VS_Snippets_CLR/math.min/CS/min.cs]: I agree. I think this obscures the purpose of the examples.

File: snippets/csharp/VS_Snippets_CLR_System/system.Math.Abs/cs/Abs1.cs (hunk 4 above)
Comment (BillWagner) [reply snippets/csharp/VS_Snippets_CLR_System/system.Math.Abs/cs/Abs1.cs]: This set of changes is optional, but highly recommended.

File: snippets/csharp/VS_Snippets_CLR/math.min/CS/min.cs (hunk 3 above)
Comment (WilliamAntonRohm) [reply snippets/csharp/VS_Snippets_CLR/math.min/CS/min.cs]: Done.

File: snippets/csharp/VS_Snippets_CLR/math.max/CS/max.cs (hunk 1 above)
Comment (WilliamAntonRohm) [reply snippets/csharp/VS_Snippets_CLR/math.max/CS/max.cs]: Done.

File: snippets/csharp/VS_Snippets_CLR_System/system.Math.Abs/cs/Abs1.cs (hunk 4 above)
Comment (WilliamAntonRohm) [reply snippets/csharp/VS_Snippets_CLR_System/system.Math.Abs/cs/Abs1.cs]: This instance is fixed -- looking for others now.

Comment (WilliamAntonRohm) [reply snippets/csharp/VS_Snippets_CLR_System/system.Math.Abs/cs/Abs1.cs]: @BillWagner, @Youssef1313 -- there are 13 more such constructs in this set of example files; working on them.

**Comment by WilliamAntonRohm** [comment]
@BillWagner, @Youssef1313 -- I've updated 11 of the 13 remaining occurrences.  Those two seem simpler as-is:

*max.cs*
```
string str = "{0}: The greater of {1,3} and {2,3} is {3}.";
...
Console.WriteLine(str, "Byte   ", xByte1, xByte2, Math.Max(xByte1, xByte2));
Console.WriteLine(str, "Int16  ", xShort1, xShort2, Math.Max(xShort1, xShort2));
Console.WriteLine(str, "Int32  ", xInt1, xInt2, Math.Max(xInt1, xInt2));
...
```
*min.cs* likewise.

**Pull request merged**